### ISSUE TITLE
python plot_importance without feature name when using np.array for training data 

### ISSUE THREAD

**Issue by machineCYC** [dmlc/xgboost#5210]
Below code, when trax is dataframe, xgb.plot_importance will show feature name, but trax is np.array it will show f1, f2, f3...

It is not convenience, could add a parameter in xgb.plot_importance call column_names(maybe other names) for transforming the  "f1, f2, f3..." to the real column name

The xgb.plot_importance is in https://github.com/dmlc/xgboost/blob/master/python-package/xgboost/plotting.py 

    import xgboost as xgb
    import matplotlib.pyplot as plt

    Xgb = xgb.XGBRegressor(n_estimators=100, max_depth=4, seed=16, min_child_weight=100, 
    gamma=0.5, reg_alpha=0.5, reg_lambda=0.5)
    Xgb.fit(trax, tray)

    xgb.plot_importance(Xgb, max_num_features=20)
    plt.show()

**Comment by arrayslayer** [comment]
Hi,
Maybe you could just do-
Xgb.fit(pd.DataFrame(trax,columns=colum _names),tray)

I think the level of convenience would be similar since you do have to provide the list of column names anyway.

**Comment by machineCYC** [comment]
I create a pull request #5761

**Comment by trivialfis** [comment]
See https://github.com/dmlc/xgboost/pull/5761#issuecomment-1085223057 .
